build.gradle

```diff
@@ -153,7 +158,22 @@ task renameShadowJar(type: Copy, group: 'release', dependsOn: [shadowJar]) {
 task updateAssets(dependsOn: ['gitCloneAssets', 'gitPullAssets']) {
 }
 
-task allPlatform(type: Zip, group: 'release', dependsOn: [renameShadowJar, updateAssets]) {
+task prepareGameEngineProperties(group: 'release') {
```

Comment:
I added a task description in ddb0dcc.  Please let me know if it needs further clarification.